src/Layers/xrRenderPC_R1/FStaticRender_Loader.cpp

```diff
@@ -425,6 +425,7 @@ void CRender::LoadSectors(IReader* fs)
 
             if (do_rebuild)
             {
+                CL.reserve_faces(P.vertices.size() - 2);
```

Comment:
I have a small suspect, that using `size() - 2` is wrong too, but not sure about this. Will check it a bit later..